Add missing tokenizer tests - RemBert

IMvision12 opened the pull request:
# What does this PR do?

Added tokenizer tests for Rembert. I took reference from `test_tokenization_camembert.py`

@SaulLu @LysandreJik

IMvision12 committed:
Create test_tokenization_rembert.py

IMvision12 committed:
Create test_tokenization_rembert.py

IMvision12 committed:
Create test_tokenization_rembert.py

IMvision12 committed:
Update test_tokenization_rembert.py

IMvision12 committed:
Update test_tokenization_rembert.py

IMvision12 committed:
Update test_tokenization_rembert.py

IMvision12 committed:
Update test_tokenization_rembert.py

IMvision12 committed:
Update test_tokenization_rembert.py

IMvision12 committed:
Update test_tokenization_rembert.py

IMvision12 committed:
Update test_tokenization_rembert.py

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_20350). All of your documentation changes will be reflected on that endpoint.

IMvision12 committed:
Update test_tokenization_rembert.py

IMvision12 committed:
Update test_tokenization_rembert.py

IMvision12 committed:
Update test_tokenization_rembert.py

IMvision12 committed:
Update test_tokenization_rembert.py

gante committed:
Generate: add generation config class (#20218)

Co-authored-by: Sylvain Gugger <35901082+sgugger@users.noreply.github.com>

Co-authored-by: Sylvain Gugger <35901082+sgugger@users.noreply.github.com>

bfss committed:
translate zh quicktour(#20095) (#20181)

* zh quicktour(#20095)

* add zh to doc workflow

* remove untranslation from toctree

Co-authored-by: BeifangSusu <BeifangSusu@bfss.com>

donelianc committed:
Add Spanish translation of serialization.mdx (#20245)

* Update _toctree and clone original content

* Translate first three sections

* Add more translated chapters. Only 3 more left.

* Finish translation

* Run style from doc-builder

* Address recommended changes from reviewer

alihassanijr committed:
Add LayerScale to NAT/DiNAT (#20325)

* Add LayerScale to NAT/DiNAT.

Completely dropped the ball on LayerScale in the original PR (#20219).
This is just an optional argument in both models, and is only activated for larger variants in order to provide training stability.

* Add LayerScale to NAT/DiNAT.

Minor error fixed.

Co-authored-by: Ali Hassani <ahassanijr@gmail.com>

IMvision12 committed:
Update test_tokenization_rembert.py

IMvision12 committed:
Update test_tokenization_rembert.py

IMvision12 committed:
Update test_tokenization_rembert.py

younesbelkada committed:
[Switch Transformers] Fix failing slow test (#20346)

* run slow test on GPU

* remove unnecessary device assignment

* use `torch_device` instead

rajrajhans committed:
fix: "BigSicence" typo in docs (#20331)

hollance committed:
add MobileNetV1 model (#17799)

* add model files etc for MobileNetV2

rename files for MobileNetV1

initial implementation of MobileNetV1

fix conversion script

cleanup

write docs

tweaks

fix conversion script

extract hidden states

fix test cases

make fixup

fixup it all

remove main from doc link

fixes

fix tests

fix up

use google org

fix weird assert

* fixup

* use google organization for checkpoints

IMvision12 committed:
Merge branch 'missing' of github.com:IMvision12/transformers into missing

IMvision12 committed:
Merge branch 'missing' of github.com:IMvision12/transformers into missing

sgugger commented:
Looks like something went wrong and GitHub adds a lot of diff. If force-pushing doesn't solve the issue, you might need to close this PR and open a fresh one.
